packages/applications/ssr/src/components/pages/puissance/modifier/ModifierPuissance.form.tsx

```diff
@@ -0,0 +1,78 @@
+'use client';
+
+import { FC, useState } from 'react';
+import Button from '@codegouvfr/react-dsfr/Button';
+import Input from '@codegouvfr/react-dsfr/Input';
+
+import { Routes } from '@potentiel-applications/routes';
+import { IdentifiantProjet } from '@potentiel-domain/common';
+import { Puissance } from '@potentiel-domain/laureat';
+import { PlainType } from '@potentiel-domain/core';
+
+import { Form } from '@/components/atoms/form/Form';
+import { SubmitButton } from '@/components/atoms/form/SubmitButton';
+import { ValidationErrors } from '@/utils/formAction';
+
+import { modifierPuissanceAction, ModifierPuissanceFormKeys } from './modifierPuissance.action';
+
+export type ModifierPuissanceFormProps = PlainType<Puissance.ConsulterPuissanceReadModel>;
+
+export const ModifierPuissanceForm: FC<ModifierPuissanceFormProps> = ({
+  identifiantProjet,
+  puissance,
+}) => {
+  const [validationErrors, setValidationErrors] = useState<
+    ValidationErrors<ModifierPuissanceFormKeys>
+  >({});
+
+  return (
+    <Form
+      action={modifierPuissanceAction}
+      onValidationError={(validationErrors) => setValidationErrors(validationErrors)}
+      actions={
+        <>
+          <Button
+            priority="secondary"
+            linkProps={{
+              href: Routes.Projet.details(IdentifiantProjet.bind(identifiantProjet).formatter()),
+              prefetch: false,
+            }}
+            iconId="fr-icon-arrow-left-line"
+          >
+            Retour à la page projet
+          </Button>
+          <SubmitButton>Modifier</SubmitButton>
+        </>
+      }
+    >
+      <input
+        name="identifiantProjet"
+        type="hidden"
+        value={IdentifiantProjet.bind(identifiantProjet).formatter()}
+      />
+
+      <div className="flex flex-col gap-6">
+        <Input
+          state={validationErrors['puissance'] ? 'error' : 'default'}
+          stateRelatedMessage={validationErrors['puissance']}
+          label="Puissance (en MWc)"
+          nativeInputProps={{
+            name: 'puissance',
+            defaultValue: puissance,
+            required: true,
+            'aria-required': true,
+          }}
```

Comment:
Vu qu'on s'attend à un nombre je rajouterais bien les props de l'input pour les number.
Cf : https://components.react-dsfr.codegouv.studio/?path=/docs/components-input--input-type-number

```suggestion
        <Input
          state={validationErrors['puissance'] ? 'error' : 'default'}
          stateRelatedMessage={validationErrors['puissance']}
          label="Puissance (en MWc)"
          nativeInputProps={{
            name: 'puissance',
            defaultValue: puissance,
            required: true,
            'aria-required': true,
            inputMode: 'numeric',
            pattern: '[0-9]*',
            type: 'number',
          }}
```

Reply:
Tu as testé ? ça marche bien ? 
On le fait pas dans les autres inputs

Reply:
Yes je viens de tester là ça devrait bien marcher je refais un dernier test